### PR TITLE
Add Workplace search docs

### PR DESCRIPTION
**Pull request by yakhinvadim**
This PR adds two new pages to our docs:
Workplace Search tutorial - https://search-c0h0a54g5-elastic-dev.vercel.app/tutorials/workplace-search
Workplace Search connector - https://search-c0h0a54g5-elastic-dev.vercel.app/connectors-api/workplace-search-connector

The tutorial was written from scratch, but the connector page was mostly copied from the existing connector readme: https://github.com/elastic/search-ui/tree/master/packages/search-ui-workplace-search-connector

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _search-ui_ successfully!

* [View deploy preview](https://search-8dit3kkfi-elastic-dev.vercel.app)

* Built with commit b171d858d6d663d32af9e374ffd8da8168bd1a19

> Issues? Visit #next-docs in Slack

**Comment by JasonStoltz** [comment]
Workplace_Search___Elastic_Docs

We don't use this same "Technical Preview" treatment for Elasticsearch connector. We should be consistent as they're both in the same Technical Preview status.

File: docs/search-ui/guides-workplace-search.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+## Creating a search page
+
+Have you indexed data into Workplace Search? Great! Let's create a search page in your application.
+
+We'll use React components in this tutorial, but you can use Search UI with any UI framework and even vanilla JavaScript. Read our <DocLink id="guides-philosophy" /> to learn more about using Search UI outside of React.
+
+First, let's install Search UI packages:
+
+```shell
+npm install @elastic/search-ui @elastic/react-search-ui @elastic/react-search-ui-views @elastic/search-ui-workplace-search-connector
```
Comment (JasonStoltz) [inline docs/search-ui/guides-workplace-search.mdx:36]: Just these two will do. The other dependencies are transitive.

```suggestion
npm install @elastic/react-search-ui @elastic/search-ui-workplace-search-connector
```

File: docs/search-ui/guides-workplace-search.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+![endpoints](images/workplace-search-tutorial/endpoints.png)
+
+### `redirectUri`
+
+Use the path to the page where Search UI is going to live.
+
+### `clientId`
+
+To get a clientID, you need to create a new OAuth application in Workplace Search.
```
Comment (JasonStoltz) [inline docs/search-ui/guides-workplace-search.mdx:109]:
```suggestion
To get a clientId, you need to create a new OAuth application in Workplace Search.
```

**Review by JasonStoltz — APPROVED** [review]
Approving, but I did leave some comments and suggestions for you to consider first.

File: docs/search-ui/guides-workplace-search.mdx (hunk 1 above)
Comment (yakhinvadim) [reply docs/search-ui/guides-workplace-search.mdx:36]: Oh, awesome, good to know.

**Comment by yakhinvadim** [comment]
I added the technical preview warning to all WS and Elasticsearch guides and readmes in this PR: https://github.com/elastic/search-ui/pull/709

**Pull request merged**